### DISCUSSION TITLE
[Please HELP] Invalid input on Endpoint /feeds/2021-06-30/documents

### DISCUSSION THREAD

**Discussion by Kinggrass** [amzn/selling-partner-api-models#3167] [General] [answered]
Hello everyone,

hopefully someone could help me or having a clue.

I am sending following request to the Endpoint:

![image](https://user-images.githubusercontent.com/11460260/137447277-c5cb99ff-3339-413d-aff4-13fee99a27d3.png)

Short for Search Engine: 

```
{
  "contentType": "text/xml; charset=UTF-8"
}
```

and getting back:
 
![image](https://user-images.githubusercontent.com/11460260/137447424-d253a212-2acf-4fa3-a39a-227c8b872166.png)

Short for Search Engine: 

`{"errors":[{"code":"400","message":"Invalid input","details":"Invalid input"}]}`

I don't get what should be wrong. I have checked all credentials and again against the documentation.

I should get back a feedDocumentId and an url.

I am trying to follow this guide:

[https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/use-case-guides/feeds-api-use-case-guide/feeds-api-use-case-guide_2021-06-30.md#step-1-create-a-feed-document](https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/use-case-guides/feeds-api-use-case-guide/feeds-api-use-case-guide_2021-06-30.md#step-1-create-a-feed-document)

**Retrieving Orders is working, for those who will question if anything else works.**

**Comment by Kinggrass** [comment, marked as answer]
Figured it out myself.

I am trying to use the "Sandbox". Per definition a place where you can try things out and they will work as expected and are then deleted after a period of time without a impact on Production.

But the Sandbox is only expecting predefined input which makes it useless as a "Sandbox" and is more a Test-Mode if the endpoints are reachable rather than to try things out.

Expected input in the sandbox mode can be found in the swagger files:
[https://github.com/amzn/selling-partner-api-models](https://github.com/amzn/selling-partner-api-models)

**Comment by ghost** [comment]
Did you find a real sandbox environnent or only the mock api ? I'm afraid I'll have to test in production
